tests/fixtures/logs.py

```diff
@@ -24,3 +28,82 @@ def gelf_logger():
     logger.addHandler(handler)
 
     return logger
+
+
+def server_event(event_number):
+    """Returns event_number of events of event_type server"""
+    return pd.DataFrame(
+        data=[
+            {
+                "username": FAKE.profile().get("username"),
+                "event_type": FAKE.uri_path(),
+                "ip": FAKE.ipv4_public(),
+                "agent": FAKE.user_agent(),
+                "host": FAKE.hostname(),  # not really as the host is the docker container ID
+                "referer": FAKE.url(),
+                "accept_language": FAKE.locale(),  # could be also something like "en-US,en;q=0.5"
+                "event": '{"POST": {}, "GET": {}}',
+                "event_source": "server",
+                "context": {
+                    "course_user_tags": {},
+                    "user_id": FAKE.random_digit_or_empty(),
+                    "org_id": FAKE.word(),
+                    "course_id": "{}+{}+{}".format(
+                        FAKE.word(), FAKE.word(), FAKE.word()
+                    ),
+                    "path": FAKE.uri_path(),  # "event_type" and "context:path" share the same URI
+                },
+                "time": FAKE.iso8601(),
+                "page": "null",
+            }
+            for i in range(event_number)
+        ]
+    )
+
+
+def browser_event(event_number):
```

Comment:
I would prefer factory_boy semantic and use `size` instead. WDYT?

```suggestion
def browser_event(size):
```

Reply:
Good Idea, I was not aware of factory_boy semantic